### ISSUE TITLE
Weirdness with exit door

### ISSUE THREAD

**Issue by rainbowskydancer** [DarkholmeTenk/TardisModIssues#411]
Hi,

Issue is: I install Aroma1997's dimensional world (as I wanted a mining world). I go to enter the TARDIS, then later when done i go to exit. I can't I'm stuck inside. Dunno why. I remove the mod, thinking there's some weird conflicting code or such, since I saw the server end load and unload dimension 6 (which is what that mod uses by default). I not only remove the mod, which didn't help me exit, but I restore from a backup, and still the same can't get out. I tried tweaking the door itself, moving to different dimensions, even to DIM6 when it existed (before I removed the mod). I tried the same dim hopping after, still can't. It took me a /tardistp 0 to exit, and it dumped me near 0,0 in DIM0. Luckily spawn isn't too far, and that game's PC has an angel wing, so it can fly.

It gets weirder. I try TPing around just to see - it helped me to fix another problem with a previous TARDIS I had: the console got corrupted due to my spawning in the large manual multiblock (taught me a lesson there, haha ;). So I eventually got to make a _new_ TARDIS (DIM4). But 2 things: I see "Tardis Interior" gets loaded twice (DIM3 is empty, no region files exist). Probably nothing.

I tried /tardisrepair and refresh, reload etc. Nothing unblocked it. Not only that, but whatever is going on, it makes it so the environmental sensor doesn't return any data when I right click on it. When it works, it works as expected, it spits out my coords, etc.

I was going to try to "recall" it to try to make it budge, so it somehow resets its surroundings and makes it so I can come and go as I please, like it usually does. But I haven't been able to do that -- this is where the TPing around comes in -- my surroundings are all dark, like in the void, but not quite, I can still see my waypoints, and the oevrhead map shows me the terrain, but dark (dunno if that was in its "night mode" but I doubt it, I couldn't see lighted areas).

**Comment by rainbowskydancer** [comment]
OK, now all at once, it's working. I managed to move it around twice, and get in and out, with no issues. I forgot to mention that it was sitting on a landing pad, whereas the 2 times I moved it -- just now, I just quit the client minutes ago -- it was near 0,0 the first time, no pad, and second, near my base, again no pad. I didn't go in the first move, second one I did.

**Comment by rainbowskydancer** [comment]
I just figured it out. It thinks the sides are obstructed, but they aren't. I have carpet down around it. They're microblock cover sized and smaller even, but it thinks the sides are blocked. Which is a pretty common issue with carpet actually.

I'm going to close this, unless you figure out a way around it :).

**Comment by rainbowskydancer** [comment]
The weirdness is back :(.

**Comment by DarkholmeTenk** [comment]
Deleting the dimension region won't stop it from getting loaded.
What exactly is happening, because I suspect this may be a duplicate of #412

**Comment by Webber88** [comment]
my exit door isnt colored at all... is that supposed to happen

**Comment by FoxPotato** [comment]
Depends what you mean by not coloured. because the inside is supposed to be grey.

**Comment by DarkholmeTenk** [comment]
No response in 2 weeks, assuming fixed.

**Comment by rainbowskydancer** [comment]
(Replying to @DarkholmeTenk's comment from 26 days ago)

Lemme check #412 and see. Could be.

Edit: pretty much exactly that, yes, thanks. I'll try the very latest darkcore in my other save (trying out Vagrim's Chance modpack ATM). I'm also running tardisMod from 25-12-2015. I stick to running that one, and not the similar releases from your jenkins instance because I don't want to lose content, like those upgrades, my TARDIS from that save depends on them right now.

Thanks for your time.

Oh, donations, I imagine you have that somewhere? :)
